TrackletAlgorithm/MemoryTemplateBinnedCM.h

```diff
@@ -199,10 +190,10 @@ class MemoryTemplateBinnedCM{
 
     DataType data("0",16);
     for (size_t ibx=0; ibx<(kNBxBins); ++ibx) {
-      for (size_t icopy=0; icopy < NCOPY; icopy++) {
-	for (size_t ibin=0; ibin < kNMemDepth; ibin++) {
-	  dataarray_[icopy][ibx][ibin] = data;
-	}
+      for (size_t icopy=0; icopy < NCP; icopy++) {
+       	for (size_t ibin=0; ibin < kNMemDepth; ibin++) {
+	        dataarray_[icopy][ibx][ibin] = data;
+      	}
```

Comment:
@tomalin Same comment regarding indentation.

```suggestion
      for (size_t icopy=0; icopy < NCP; icopy++) {
        for (size_t ibin=0; ibin < kNMemDepth; ibin++) {
          dataarray_[icopy][ibx][ibin] = data;
        }
```